.circleci/config.yml

```diff
@@ -14,9 +14,9 @@ references:
   restore_cache: &restore_cache
     restore_cache:
       keys:
-        - deps-v2-{{ .Branch }}-{{ checksum "yarn.lock" }}
-        - deps-v2-{{ .Branch }}
-        - deps-v2
+        - deps-v4-{{ .Branch }}-{{ checksum "yarn.lock" }}
```

Comment:
What does v4 mean here?

Reply:
It's a poor man's way of forcing circle to drop all cached `node_modules`.
Before, I got these weird errors were some babel plugin did not install correctly (which is usually fixed by nuking node_modules): https://app.circleci.com/pipelines/github/signavio/react-stick/2707/workflows/e162e001-8ddb-4765-bd58-8bd9878d7faf/jobs/11725